spec/controllers/admin/admin_users_controller_spec.rb

```diff
@@ -13,5 +13,49 @@
         expect(response.status).to eq(200)
       end
     end
+
+    describe "GET /admin/users/edit/1" do
```

Comment:
__[RuboCop]__

Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.


<!-- Comment by [SideCI](https://sideci.com) -->